src/Jackett.Common/Indexers/FileListApi.cs

```diff
@@ -0,0 +1,216 @@
+            {
+                queryCollection.Add("type", "name");
+                queryCollection.Add("query", searchString);
+                queryCollection.Add("action", "search-torrents");
+            } 
+            else 
+            {
+                queryCollection.Add("action", "latest-torrents");
+            }
+
+            queryCollection.Add("category", cat);
+
+
+            searchUrl += "?" + queryCollection.GetQueryString();
+
+            var response = await RequestStringWithCookiesAndRetry(searchUrl, null, null, GetAuthHeader());
+
+            return response.Content;
+        }
+
+        private List<ReleaseInfo> TransformProviderRequest(TorznabQuery query, string results)
+        {
+            var releases = new List<ReleaseInfo>();
+            JArray json = null;
+
+            try
+            {
+                json = JArray.Parse(results);
+                foreach (JObject r in json)
+                {
+                    var link = new Uri((string)r["download_link"]);
+                    var seeders = (int)r["seeders"];
+                    var publishDate = DateTimeUtil.FromFuzzyTime((string)r["upload_date"]);
+                    var release = new ReleaseInfo
+                    {
+                        Title = (string)r["name"],
+                        Comments = null,
+                        Link = link,
+                        Category = MapTrackerCatDescToNewznab((string)r["category"]),
+                        Size = (long)r["size"],
+                        Files = (long)r["files"],
+                        Grabs = (long)r["times_completed"],
+                        Seeders = seeders,
+                        PublishDate = publishDate,
+                        UploadVolumeFactor = 1,
+                        DownloadVolumeFactor = GetDownloadFactor(r),
+                        Guid = link,
+                        Peers = seeders + (int)r["leechers"]
+                    };
+
+                    if (r.ContainsKey("imdb"))
+                    {
+                        release.Imdb = ParseUtil.GetImdbID((string)r["imdb"]);
+                    }
+
+                    releases.Add(release);
+                }
+                return releases;
+            }
+            catch (Exception ex)
+            {
+                OnParseError(results, ex);
+            }
+            return null;
+        }
+
+        private Dictionary<string, string> GetAuthHeader()
+        {
+            var auth = Convert.ToBase64String(Encoding.UTF8.GetBytes(configData.Username.Value + ":" + configData.Passkey.Value));
+            var headers = new Dictionary<string, string>()
+            {
+                { "Authorization", "Basic " + auth }
+            };
+            return headers;
+        }
+
+        private static double GetDownloadFactor(JObject r)
+        {
+            // 100% Free Leech: all blue torrents.
+            if ((int)r["freeleech"] == 1)
```

Comment:
I think you don't need this method.

var dlVolumeFactor = (int)r["freeleech"] == 1 ? 0 : 1;